src/panel_material_ui/base.py

```diff
@@ -338,3 +350,51 @@ def controls(self, parameters: list[str] = None, jslink: bool = True, **kwargs)
             return Tabs(controls.layout[0], style.layout[0])
         elif params:
             return controls.layout[0]
+
+class CustomMaterialComponent(MaterialComponent):
```

Comment:
I would suggest a name change suggest that I can use something 1) compliant with Panel custom components and 2) enabling other Material component libraries in the future:

```python
from panel_material_ui.custom import MaterialUIComponent
```

similar to

```python
from panel.custom import ReactComponent
```

Reply:
Sure, this PR is not anywhere near where I'm concerned about naming.

Reply:
I got it working so this is no longer theoretical.